helm/aladdin-demo/templates/server/deploy.yaml

```diff
@@ -47,71 +45,77 @@ spec:
           - mountPath: /config/uwsgi.yaml
             name: {{ .Chart.Name }}-uwsgi
             subPath: uwsgi.yaml
-{{ if .Values.deploy.withMount }}
-          - mountPath: /home
+          {{ if .Values.deploy.withMount }}
+          - mountPath: /home/app
             name: {{ .Chart.Name }}-server
-            subPath: app
-{{ end }} # /withMount
+          {{ end }}
         envFrom:
           # Load the data from configMap into the runtime environment
           # This allows us to use os.environ["KEY"] to look up variables
           - configMapRef:
               name: {{ .Chart.Name }}
+        {{ if .Values.resourceLimits.enable }}
         resources:
           requests:
-            cpu: 100m
-            memory: 100Mi
+            cpu: {{ .Values.resourceLimits.server.uwsgi.cpu.requests }}
+            memory: {{ .Values.resourceLimits.server.uwsgi.memory.requests }}
           limits:
-            cpu: 200m
-            memory: 200Mi
-        {{ if .Values.app.readiness.use }}
```

Comment:
moved readiness and liveness to the nginx section - readiness checks nginx and liveness checks uwsgi via nginx